test/integration/iam/statistics/test_sparql_statistics_with_iam.py

```diff
@@ -0,0 +1,80 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+"""
+import pytest
+from botocore.session import get_session
+
+from test.integration import IntegrationTest
+
+
+class TestSparqlStatisticsWithIAM(IntegrationTest):
```

Comment:
Can we use parameterized tests with 'pg' & 'sparql' arguments to simplify the integration tests since the tests in this class are same as `test/integration/iam/statistics/test_pg_statistics_with_iam.py`?

Thinking of something like [Junit's @ValueSource](https://www.baeldung.com/parameterized-tests-junit-5#1-simple-values) for python.

Reply:
Merged PG and Sparql test classes to single one with language parameters.